<launch>
  <arg name="param_file" default="$(find-pkg-share autoware_diagnostic_graph_aggregator)/config/default.param.yaml"/>
  <arg name="graph_file"/>
  <node pkg="autoware_diagnostic_graph_aggregator" exec="aggregator_node">
    <param from="$(var param_file)"/>
    <param name="graph_file" value="$(var graph_file)"/>
  </node>
  <node pkg="autoware_diagnostic_graph_aggregator" exec="converter_node">
    <param from="$(find-pkg-share autoware_diagnostic_graph_aggregator)/config/converter.param.yaml"/>
    <remap from="~/command_mode/availability" to="/system/command_mode/availability"/>
    <remap from="~/operation_mode/availability" to="/system/operation_mode/availability"/>
  </node>
</launch>
